### PR TITLE
Fix Travis CI, coverage & coveralls

### PR DESCRIPTION
**Pull request by AnthonyHorton**
Get continuous integration working properly for `huntsman-pocs`. Mostly this is a case of copying parts of the relevant config files from `panoptes/POCS`.  First few commits cherry picked from the (unrelated) PR #75

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,21 +6,28 @@ services:
...
+  - PANDIR=/var/huntsman POCS={$PANDIR}/POCS PANUSER=$USER HUNSTMAN_POCS={$PANDIR}/huntsman-pocs
 before_install:
-    - sudo mkdir /var/huntsman && sudo chmod 777 /var/huntsman
-    - cd /var/huntsman
-    - git clone https://github.com/huntsman/POCS.git
-    - cd POCS
-    - pip install -r requirements.txt
-    - python setup.py install
-    - ln -s $TRAVIS_BUILD_DIR /var/huntsman/POCS
+  - sudo mkdir /var/huntsman && sudo chmod 777 /var/huntsman
```
Comment (wtgee) [inline .travis.yml]: Could go ahead and replace all the `/var/huntsman` with `PANDIR`.

File: matplotlibrc [hunk 2]
```diff
@@ -0,0 +1 @@
+backend: Agg
```
Comment (wtgee) [inline matplotlibrc:1]: New matplotlib > 3 supposedly makes all this unnecessary: https://matplotlib.org/users/whats_new.html

**Review by wtgee — APPROVED** [review]
LGTM

File: matplotlibrc (hunk 2 above)
Comment (AnthonyHorton) [reply matplotlibrc:1]: Adding this did seem to remove a bunch of warnings from `matplotlib`, though.

**Comment by AnthonyHorton** [comment]
importlib!

File: .travis.yml (hunk 1 above)
Comment (AnthonyHorton) [reply .travis.yml]: OK.

File: matplotlibrc (hunk 2 above)
Comment (AnthonyHorton) [reply matplotlibrc:1]: May be because `gunagala` is doing something wrong.

**Comment by AnthonyHorton** [comment]
Want to get this merged but need fix that importlib error first. No idea what that's about!

**Comment by AnthonyHorton** [comment]
Merging this now so we can track progress on #77

**Pull request merged**